packages/back-end/src/controllers/experimentLaunchChecklist.ts

```diff
@@ -135,9 +134,7 @@ export async function putManualLaunchChecklist(
     throw new Error("Could not find experiment");
   }
 
-  const envs = experiment ? getAffectedEnvsForExperiment({ experiment }) : [];
-
-  if (!context.permissions.canRunExperiment(experiment, envs)) {
+  if (!context.permissions.canViewExperimentModal(experiment.project)) {
```

Comment:
This seems like it should be `canUpdateExperiment` instead, right?

Reply:
DOH. Yes. I've updated the both places to use the `canUpdateExperiment`.